python_tests/pytest_framework.py

```diff
@@ -53,5 +54,11 @@ def run_test_fn(test_fn, opinfo, dtype, *args, **kwargs):
     try:
         test_fn(*args, **kwargs)
     except Exception as e:
+        # Raises exceptions that occur with pytest, and returns debug information when
```

Comment:
Small fix for raising exception in test.

Reply:
dumb question, is this supposed to be used out-side of pytest contest? It doesn't look clear to me why we have diverging behavior in `pytest_framework.py` :confused:

Reply:
I removed `run_test_fn` since we aren't using this outside of `pytest`